### DISCUSSION TITLE
Implement Tailwind Intellisense on appropriate props

### DISCUSSION THREAD

**Discussion by ryceg** [skeletonlabs/skeleton#505] [Ideas]
### Describe what feature you'd like. Pseudo-code, mockups, or screenshots of similar solutions are encouraged!

So, as we all know the Tailwind Intellisense only procs on `class` and `className` properties. Might be worth digging through the Tailwind extension source code to see if we can figure out a way to get it to proc on the properties that we want. 

An alternative is, of course, to construct types that do kinda what we want; union types could do a facsimile of this.

```ts
	type TailwindNumbers = '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9'
	type TailwindLevel = 'primary' | 'accent' | 'ternary' | 'warning' | 'surface'
	type TailwindColors = `${TailwindLevel}-${TailwindNumbers}00` | string;
```
gets us all of the Tailwind colors. For some things, this sort of faux-intellisense could be a boon, but would not work for everything.

### Any links to similar examples or other references we should review?

https://github.com/tailwindlabs/tailwindcss-intellisense

**Comment by endigo9740** [comment]
FYI The tailwind color shades start at `50`!

**Comment by endigo9740** [comment]
@ryceg I'm doing some house keeping and going to go ahead and close this out. Just a reminder that the Discussion section has a category called "ideas" that might be more apt for posts like this. The idea is good, just not something we may act on soon.

**Comment by endigo9740** [comment]
Just noticed I have a "convert to discussion" option so I'm going to do that!
